Palindrome.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace PalindromeArray
+{
+    public class Palindrome
+    {
+        public bool IsArrayPalindrome(char[] arr)
+        {
+            if (arr.Length == 0)
+            {
+                return false;
+            }
+
+            int start = 0;
+            int end = arr.Length - 1;
+
+            while (start <= end)
+            {
+                if (arr[start] == arr[end])
+                {
+                    start++;
+                    end--;
+                }
+                else
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+        public char[] PalindromeHelper(string s)
```

Comment:
A couple of items to consider about this method (some apply to interviewing and others just to regular coding):

1. Why is it public? Do you want your users to call it?
2. The method name should do a better job describing what you are doing here. Perhaps something like "ConvertToCharArray" or "ConvertToCleanCharArray".